src/main/java/org/example/lesson4/juma/homework/Calculator.java

```diff
@@ -0,0 +1,54 @@
+package org.example.lesson4.juma.homework;
+
+import java.util.Scanner;
+
+public class Calculator {
+
+
+
+    public double num1;
+    public double num2;
```

Comment:
Include a constructor that allows you to set these properties when creating an object.